# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xfsprogs/hotfix-madvice.patch.mips64
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xfsprogs-6.8.0/io/madvise.c.vanilla	2024-07-25 20:39:23.804908156 +0200
+++ xfsprogs-6.8.0/io/madvise.c	2024-07-25 20:40:43.390907195 +0200
@@ -10,6 +10,7 @@
 #include "init.h"
 #include "io.h"
 #include <asm/mman.h>
+#include <asm-generic/mman.h>
 
 static cmdinfo_t madvise_cmd;
 
